### DISCUSSION TITLE
how to delete all .next/cache/images

### DISCUSSION THREAD

**Discussion by ngocchu153** [vercel/next.js#41589] [Help]
Context: I have a NextJS project using next/image and deployed it on Vercel. Now I updated some images on AWS S3, I want to clear all the cached images on the server. 

I have searched around, there is no way to ssh into the server. How can I delete the cached folder?

**Comment by icyJoseph** [comment]
I think you have to redeploy without the build cache, or have a proper header on the cache control for the images, from the upstream, so that they, naturally over time are revalidated. I don't think there's a way to imperatively do it though.

**Comment by thiswallz** [comment]
.next can be ignored in your .gitignore file
